GestureHandler.js

```diff
@@ -15,46 +13,19 @@ import {
   StyleSheet,
   FlatList,
   Platform,
+  processColor,
+  Touchable,
 } from 'react-native';
-import processColor from 'react-native/Libraries/StyleSheet/processColor';
-import Touchable from 'react-native/Libraries/Components/Touchable/Touchable';
 import deepEqual from 'fbjs/lib/areEqual';
 import PropTypes from 'prop-types';
 
+import GestureHandlerButton from './GestureHandlerButton.web';
+import GestureHandlerModule from './GestureHandlerModule.web';
 import gestureHandlerRootHOC from './gestureHandlerRootHOC';
 
-const RNGestureHandlerModule = NativeModules.RNGestureHandlerModule;
-
-/* Wrap JS responder calls and notify gesture handler manager */
-const { UIManager } = NativeModules;
-const {
-  setJSResponder: oldSetJSResponder,
-  clearJSResponder: oldClearJSResponder,
-} = UIManager;
-UIManager.setJSResponder = (tag, blockNativeResponder) => {
-  RNGestureHandlerModule.handleSetJSResponder(tag, blockNativeResponder);
-  oldSetJSResponder(tag, blockNativeResponder);
-};
-UIManager.clearJSResponder = () => {
-  RNGestureHandlerModule.handleClearJSResponder();
-  oldClearJSResponder();
-};
-
-// Add gesture specific events to genericDirectEventTypes object exported from UIManager
-// native module.
-// Once new event types are registered with react it is possible to dispatch these
-// events to all kind of native views.
-UIManager.genericDirectEventTypes = {
-  ...UIManager.genericDirectEventTypes,
-  onGestureHandlerEvent: { registrationName: 'onGestureHandlerEvent' },
-  onGestureHandlerStateChange: {
-    registrationName: 'onGestureHandlerStateChange',
-  },
-};
-
-const State = RNGestureHandlerModule.State;
-
-const Directions = RNGestureHandlerModule.Direction;
+import Directions from './Directions';
+import State from './State';
+import PlatformConstants from './PlatformConstants.web';
```

Comment:
it is intendeed to import `PlatformConstants.web` ? Same with the `GestureHandlerButton` and `GestureHandlerModule`

Reply:
Oops, good catch. Seems that VS code automatically changed the imports when I renamed the files. Will change it on Monday latest.